resources/services/kinesis/gen.hcl

```diff
@@ -45,9 +45,255 @@ resource "aws" "kinesis" "streams" {
+      type        = "json"
+      description = "The Amazon Resource Name (ARN) of the delivery stream"
+      resolver "resolveProcessors" {
+        path   = "github.com/cloudquery/cq-provider-aws/client.ResolvePathJson"
+        params = ["ProcessingConfiguration.Processors"]
+      }
+    }
+    column "request_configuration_common_attributes" {
+      // skip = true
+      type              = "json"
+      resolver "pathResolver" {
+        path = "github.com/cloudquery/cq-provider-sdk/provider/schema.PathResolver"
+        params = ["RequestConfiguration.CommonAttributes"]
+      }
+    }
+  }
+  user_relation "aws" "kinesis" "redshift_destination" {
+    path = "github.com/aws/aws-sdk-go-v2/service/firehose/types.RedshiftDestinationDescription"
+    resolver "resolveTable" {
+      path          = "github.com/cloudquery/cq-provider-sdk/provider/schema.PathTableResolver"
+      path_resolver = true
+      params        = ["Destinations.RedshiftDestinationDescription"]
+    }
+    column "s3_destination_encryption_configuration" {
+      skip_prefix = true
+    }
+    //s3_backup_encryption_configuration_kms_encryption_config_aws_kms_key_arn
+    // column "s3_destination_encryption_configuration_kms_encryption" {
+    //   skip_prefix = true
+    // }
+    column "s3_destination_description" {
+      rename = "s3_destination"
+    }
+    column "s3_backup_description" {
+      rename = "s3_backup"
+    }
+
+    column "s3_backup_encryption_configuration" {
+      skip_prefix = true
+    }
+    column "processing_configuration_processors" {
+      skip = true
+    }
+    userDefinedColumn "processing_configuration_processors" {
+      type        = "json"
+      description = "The Amazon Resource Name (ARN) of the delivery stream"
+      resolver "resolveProcessors" {
+        path   = "github.com/cloudquery/cq-provider-aws/client.ResolvePathJson"
+        params = ["ProcessingConfiguration.Processors"]
+      }
+    }
+  }
+  user_relation "aws" "kinesis" "splunk_destination" {
+    path = "github.com/aws/aws-sdk-go-v2/service/firehose/types.SplunkDestinationDescription"
+    resolver "resolveTable" {
+      path          = "github.com/cloudquery/cq-provider-sdk/provider/schema.PathTableResolver"
+      path_resolver = true
+      params        = ["Destinations.SplunkDestinationDescription"]
+    }
+    column "s3_destination_description" {
+      rename = "s3_destination"
+    }
+        column "s3_destination_encryption_configuration" {
+      skip_prefix = true
+    }
+
+    column "processing_configuration_processors" {
+      skip = true
+    }
+    userDefinedColumn "processing_configuration_processors" {
+      type        = "json"
+      description = "The Amazon Resource Name (ARN) of the delivery stream"
+      resolver "resolveProcessors" {
+        path   = "github.com/cloudquery/cq-provider-aws/client.ResolvePathJson"
+        params = ["ProcessingConfiguration.Processors"]
```

Comment:
can't you do the
```hcl
column "processing_configuration_processors" {
      type = "json"
}
```

Reply:
but can you then just ignore it in tests?
or tweak the data a bit?

Reply:
When you ignore it in tests, then that means that even if data is present in the struct it will never be persisted to the database.

What type of tweaking to the data would you like to see?